ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/dot1x.md

```diff
@@ -111,4 +125,44 @@
 
         # Interval period in seconds.
         accounting_update_interval: <int; 5-65535>
+
+      # Web authentication feature authenticates a supplicant through a web page, referred to as a captive portal.
+      captive_portal:
+
+        # Supported URL type:
+        #   - http: http[s]://<hostname>[:<port>]
+        #   - https: http[s]://<hostname>[:<port>]
+        url: <str>
+
+        # SSL profile name.
+        ssl_profile: <str>
+
+        # Standard access-list name.
+        access_list_ipv4: <str>
+      supplicant:
+
+        # Dot1x supplicant profiles.
+        profiles:
+          - name: <str; required; unique>
+
+            # Extensible Authentication Protocol method:
+            #   - EAP Flexible Authentication via Secure Tunneling.
+            #   - EAP with Transport Layer Security.
+            eap_method: <str; "fast" | "tls">
+
+            # User identity.
+            identity: <str>
+            passphrase_type: <str; "0" | "7" | "8a"; default="7">
+
+            # Extensible Authentication Protocol password.
+            passphrase: <str>
+
+            # SSL profile name.
+            ssl_profile: <str>
+
+        # Enable supplicant logging.
+        logging: <bool>
+
+        # Timeout period in seconds.
```

Comment:
Timeout in seconds for removing a disconnected supplicant

Reply:
Fixed